config/routes.rb

```diff
@@ -4,7 +4,7 @@
       resources :pings, only: [:index], constraints: { format: 'json' }
     end
     namespace :v1 do
-      resources :articles, only: [:index, :create, :show]
+      resources :categories, :articles, only: [:index, :create, :show]
     end
```

Comment:
id say we should limit the routes open for the `:categories`, for example we wont be editing or deleting categories so those should definitely be excluded.